From f32381c605f9bff6d4cbcd1edfc0185a3bc91684 Mon Sep 17 00:00:00 2001
Message-ID: <f32381c605f9bff6d4cbcd1edfc0185a3bc91684.1699403992.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 13:43:16 +0000
Subject: [PATCH 34/67] gnu: go-github-com-go-chi-chi-v5: Move to (gnu packages
 golang-web).

* gnu/packages/golang.scm (go-github-com-go-chi-chi-v5): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I120c2cb142935430443a7e6c9e53aa8544bbaf8e
---
 gnu/packages/golang-web.scm | 27 +++++++++++++++++++++++++--
 gnu/packages/golang.scm     | 23 -----------------------
 2 files changed, 25 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index f59415452f..2570be62f7 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -1,17 +1,17 @@
 ;;; GNU Guix --- Functional package management for GNU
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
 ;;; Copyright © 2020 Martin Becze <mjbecze@riseup.net>
-;;; Copyright © 2020 Martin Becze <mjbecze@riseup.net>
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
 ;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
 ;;; Copyright © 2021, 2022 Efraim Flashner <efraim@flashner.co.il>
 ;;; Copyright © 2022 Giacomo Leidi <goodoldpaul@autistici.org>
+;;; Copyright © 2022 Sharlatan Hellseher <sharlatanus@gmail.com>
+;;; Copyright © 2022 jgart via Guix-patches via <guix-patches@gnu.org>
 ;;; Copyright © 2022 muradm <mail@muradm.net>
 ;;; Copyright © 2023 Hilton Chain <hako@ultrarare.space>
 ;;; Copyright © 2023 Katherine Cox-Buday <cox.katherine.e@gmail.com>
 ;;; Copyright © 2023 Nicolas Graves <ngraves@ngraves.fr>
-;;; Copyright © 2022 Sharlatan Hellseher <sharlatanus@gmail.com>
 ;;;
 ;;; This file is part of GNU Guix.
 ;;;
@@ -284,6 +284,29 @@ (define-public go-github-com-felixge-httpsnoop
 application's http.Handlers.")
     (license license:expat)))
 
+(define-public go-github-com-go-chi-chi-v5
+  (package
+    (name "go-github-com-go-chi-chi-v5")
+    (version "5.0.7")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/go-chi/chi")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0rzrsxz4xj0973c6nxklvq2vmg2m795snhk25836i0gnd1jnx79k"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/go-chi/chi/v5"))
+    (home-page "https://github.com/go-chi/chi")
+    (synopsis "Composable router for HTTP services written in Go")
+    (description
+     "@code{go-github-com-go-chi-chi-v5} is an HTTP router that lets the user
+decompose request handling into many smaller layers.")
+    (license license:expat)))
+
 (define-public go-github-com-google-safehtml
   (package
     (name "go-github-com-google-safehtml")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index b5f406de83..3c1693b0c8 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -11854,29 +11854,6 @@ (define-public go-github-com-dreamacro-go-shadowsocks2
 @end itemize")
     (license license:asl2.0)))
 
-(define-public go-github-com-go-chi-chi-v5
-  (package
-    (name "go-github-com-go-chi-chi-v5")
-    (version "5.0.7")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/go-chi/chi")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "0rzrsxz4xj0973c6nxklvq2vmg2m795snhk25836i0gnd1jnx79k"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/go-chi/chi/v5"))
-    (home-page "https://github.com/go-chi/chi")
-    (synopsis "Composable router for HTTP services written in Go")
-    (description
-     "@code{go-github-com-go-chi-chi-v5} is an HTTP router that lets the user
-decompose request handling into many smaller layers.")
-    (license license:expat)))
-
 (define-public go-sigs-k8s-io-yaml
   (package
     (name "go-sigs-k8s-io-yaml")
-- 
2.41.0

